gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -846,3 +846,6 @@ def finalized_outlier_filtering(test: bool = False) -> VersionedTableResource:
         for version in VERSIONS
     },
 )
+
+# Sample QC meta HT global and row annotation definitions.
+sample_qc_field_defs_json = "gs://gnomad/v4.0/sample_qc/additional_resources/gnomad.joint.v4.0.sample_qc_field_definitions.json"
```

Comment:
Maybe make it like this that I added for the JSON parameters of the trio identification (but without test): 

def ped_filter_param_json_path(version: str = CURRENT_VERSION, test: bool = False):
    """
    Get path to JSON file containing filters used to create the finalized Pedigree and trios resources.
    :param version: Version of the JSON to return.
    :param test: Whether to use a tmp path for a test resource.
    :return: Path to Pedigree filter JSON.
    """
    return f"{get_sample_qc_root(version)}/relatedness/trios/gnomad.exomes.v{version}.ped_filters.json"

Reply:
Sorry --  I see the finish line and I'm sprinting! Definitely should be a function.